privaterelay/settings.py

```diff
@@ -31,6 +31,10 @@
 SECRET_KEY = config('SECRET_KEY', None, cast=str)
 
 DEBUG = config('DEBUG', False, cast=bool)
+if DEBUG:
+    INTERNAL_IPS = [
+        '127.0.0.1',
```

Comment:
Doesn't this IP value change based on the environment? If so we should add something like`DEBUG_INTERAL_IP` var to the config instead of adding a static value here. Would be good to update the `.env-dist` as well.

Reply:
Updated.